### ISSUE TITLE
Change default email option

### ISSUE THREAD

**Issue by ryanleesipes** [thunderbird/appointment#408]
By default, Appointment uses the email associated with your Mozilla account. My Mozilla account is associated with my personal email - and I'd like to keep it that way. But I primarily use Appointment with my work email. To accept an appointment, you have to accept via the automated email, which I would like to change to go to my work email instead of the one affiliated with my Mozilla account.

TLDR: Change what email is used with Appointment without having to change the email associated with your Mozilla account.

**Comment by MelissaAutumn** [comment]
This should be easy enough, we'll need to add a preferred email address and just use that in-place of any existing subscriber.email.

**Comment by ryanleesipes** [comment]
Great, thank you Mel

**Comment by ryanleesipes** [comment]
Thanks!
